rmc/pipeline/regional_constraint.py

```diff
@@ -63,212 +45,42 @@
-
-                logger.info("Calculating expected values per transcript...")
-                exp_ht = calculate_exp_per_transcript(
-                    context_auto_ht,
-                    locus_type="autosomes",
-                    groupings=GROUPINGS,
-                )
-                exp_x_ht = calculate_exp_per_transcript(
-                    context_x_ht,
-                    locus_type="X",
-                    groupings=GROUPINGS,
-                )
-                exp_y_ht = calculate_exp_per_transcript(
-                    context_y_ht,
-                    locus_type="Y",
-                    groupings=GROUPINGS,
-                )
-                exp_ht = exp_ht.union(exp_x_ht).union(exp_y_ht)
-
-                logger.info(
-                    "Fixing expected values for genes that span PAR and nonPAR"
-                    " regions..."
-                )
-                # Adding a sum here to make sure that genes like XG that span PAR/nonPAR regions
-                # have correct total expected values
-                exp_ht = exp_ht.group_by(transcript=exp_ht.transcript).aggregate(
-                    section_exp=hl.agg.sum(exp_ht.expected),
-                    section_mu=hl.agg.sum(exp_ht.mu_agg),
-                )
-                # TODO: Write exp HT here
-
-                logger.info(
-                    "Aggregating total observed variant counts per transcript..."
-                )
-                obs_ht = calculate_observed(exome_ht)
-                # TODO: Write obs HT here
-
+            logger.info("Creating constraint prep HT...")
+            if (
+                not args.prep_missense
+                and not args.prep_nonsense
+                and not args.prep_synonymous
+            ):
+                create_constraint_prep_ht(filter_csq=False, overwrite=args.overwrite)
             else:
-                logger.warning(
-                    "Using observed and expected values calculated using LoF"
-                    " pipeline..."
-                )
-                exp_ht = (
-                    constraint_ht.ht()
-                    .key_by("transcript")
-                    .select("obs_mis", "exp_mis", "oe_mis")
-                )
-
-                # Filter to canonical transcripts only and rename fields
-                exp_ht = exp_ht.filter(exp_ht.canonical)
-                exp_ht = exp_ht.transmute(
-                    observed=exp_ht.obs_mis,
-                    expected=exp_ht.exp_mis,
-                    mu_agg=exp_ht.mu_mis,
+                csq = set()
+                if args.prep_missense:
+                    csq = csq.add(MISSENSE)
+                if args.prep_nonsense:
+                    csq = csq.add(NONSENSE)
+                if args.prep_synonymous:
+                    csq = csq.add(SYNONYMOUS)
+                create_constraint_prep_ht(
+                    filter_csq=True, csq=csq, overwrite=args.overwrite
```

Comment:
```suggestion
                    filter_csq=csq, overwrite=args.overwrite
```
(if filter_csq type is changed)